common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/TokenResponse.java

```diff
@@ -111,6 +111,23 @@
     @SerializedName("id_token")
     private String mIdToken;
 
+    private transient String mTokenAuthority;
+
+    /**
+     * Get the authority that issued this token.
+     * @return the authority that issued this token.
+     */
+    public String getAuthority() {
```

Comment:
Are there usages as of now?

Reply:
No, but this is an attempt to replace some of the circumlocution that we're going through to figure out authorities.